docs/Overview.rst

```diff
@@ -36,17 +36,67 @@ MVS Validation Plan
+*Model verification* is related to computer programming and looks into whether the code is a correct representation of the conceptual model. To accomplish this, static testing methods are used to validate the output with respect to an input. Unit tests and integration tests, using proof of correctness techniques, are integrated within the code and evaluate the output of the MVS for any change occuring as they are automated. Unit tests target a single unit such as an individual component, while integration tests target more general parts such as entire modules. Finally, it is very important to make sure that Pytest, the chosen framework for functional testing, always goes through.
 
-*Operational validity* determines if the model’s output is within the required accuracy. In order to achieve that, several validation techniques are used, namely:
+*Operational validity* assesses the model’s output with respect to the required accuracy. In order to achieve that, several validation techniques are used, namely:
 
-*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one;
+* Graphical display, which is the use of model generated or own graphs for result interpretation. Graphs are simultaneously used with other validation techniques to inspect the results;
+
+*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one to evaluate the performance of the model;
   
-*	Extreme scenarios (e.g., drastic meteorological conditions), created to make sure the simulation is through, then evaluate the output behavior by the use of graphs and qualitative analysis;
+*	Extreme scenarios (e.g., drastic meteorological conditions, very high costs, etc.), created to make sure the simulation is through and check if the output behavior is still valid by the use of graphs and qualitative analysis;
   
-*	Replication of one pilot project, with a validated optimization model with different component representation to compare and assess the results; and
+*	Comparison to other validated model, which replicates one pilot project into a validated optimization model with different component representation to identify the similarities and differences in results; and
   
 *	Sensitivity analysis, through which input-output transformations are studied to show the impact of changing the values of some input parameters.
 
+Implemented Tests
+#################
+
+Unit tests and integration tests are gauged by using test coverage measurement. The MVS covers so far 80% of the modules and sub-modules as seen in the next figure. Since those tests are automated, this coverage is updated for any changes in the model. The figure also shows that the build is currently passing.
```

Comment:
```suggestion
Unit tests and integration tests are gauged by using test coverage measurement. The MVS covers so far 80% of the modules and sub-modules as seen in the next figure. Since those tests are automated, this coverage is updated for any changes in the model.
```
You should explain that pytests are used for unit and integration tests, and that they always assert that an externally determined output is archieved when applying a specific function.

Reply:
You could also mention that the goal is to have unit tests for each single function of the MVS, and integration tests for the larger modules.